code/modules/surgery/organs/organ_external.dm

```diff
@@ -240,7 +240,7 @@
 
 	add_fingerprint(user)
 
-	var/list/implements = list(/obj/item/weapon/circular_saw, /obj/item/weapon/melee/energy/sword/cyborg/saw, /obj/item/weapon/melee/arm_blade,\
+	var/list/implements = list(/obj/item/weapon/circular_saw, /obj/item/weapon/circular_saw/bonesaw, /obj/item/weapon/melee/energy/sword/cyborg/saw, /obj/item/weapon/melee/arm_blade,\
```

Comment:
If I can be completely honest I'm not a fan of this list. Can we use any sharp object instead?